api/src/main/java/io/opentelemetry/OpenTelemetry.java

```diff
@@ -77,18 +84,47 @@ public static MeterRegistry getMeterRegistry() {
   }
 
   /**
-   * Returns a singleton {@link DistributedContextManager}.
+   * Returns a singleton {@link CorrelationContextManager}.
    *
    * @return registered manager or default via {@link
-   *     DefaultDistributedContextManager#getInstance()}.
+   *     DefaultCorrelationContextManager#getInstance()}.
    * @throws IllegalStateException if a specified manager (via system properties) could not be
    *     found.
    * @since 0.1.0
    */
-  public static DistributedContextManager getDistributedContextManager() {
+  public static CorrelationContextManager getCorrelationContextManager() {
     return getInstance().contextManager;
   }
 
+  /**
+   * Returns a {@link Propagators} object, which can be used to access the set of registered
+   * propagators for each supported format.
+   *
+   * @return registered propagators container, defaulting to a {@link Propagators} object with
+   *     {@code HttpTraceContext} registered.
+   * @throws IllegalStateException if a specified manager (via system properties) could not be
+   *     found.
+   * @since 0.3.0
+   */
+  public static Propagators getPropagators() {
+    return getInstance().propagators;
+  }
+
+  /**
+   * Sets the {@link Propagators} object, which can be used to access the set of registered
+   * propagators for each supported format.
+   *
+   * @param propagators the {@link Propagators} object to be registered.
+   * @throws IllegalStateException if a specified manager (via system properties) could not be
+   *     found.
+   * @throws NullPointerException if {@code propagators} is {@code null}.
+   * @since 0.3.0
+   */
+  public static void setPropagators(Propagators propagators) {
```

Comment:
Not sure if this needs to be here or in the SDK, what is the motivation to have it in the API? Happy to leave it here, but I want to understand the motivation (I expect that the application owner will configure this when they configure the SDK/Implementation as well).